patches/chrome-android-BUILD.gn.patch

```diff
@@ -11,3 +19,12 @@ index 02f17d2b4e93d27964fe6663c4feddccc9634619..bd0459db766702d7bc0127914e687421
  test_manifest_package = "org.chromium.chrome.tests"
  
  chrome_public_jinja_variables = default_chrome_public_jinja_variables +
+@@ -412,7 +413,7 @@ android_library("chrome_java") {
+   android_manifest_for_lint = chrome_public_android_manifest
+ 
+   # From java_sources.gni.
+-  java_files = chrome_java_sources + [
```

Comment:
I think it's better to do `java_files += brave_java_sources` on a separate line

Reply:
re https://github.com/brave/brave-core/pull/2604/files#r290988833 maybe this should be `chrome_java_sources += brave_java_sources` in java_sources.gni?

Reply:
done